omreader/src/Main.scala

```diff
@@ -0,0 +1,56 @@
+// SPDX-License-Identifier: Apache-2.0
+// SPDX-FileCopyrightText: 2022 Jiuyang Liu <liu@jiuyang.me>
+
+package org.chipsalliance.t1.omreader
+
+import java.io.BufferedInputStream
+import mainargs._
+import chisel3.panamaom._
+import org.chipsalliance.t1.omreaderlib._
+
+object Main {
+  implicit object PathRead extends TokensReader.Simple[os.Path] {
+    def shortName = "path"
+    def read(strs: Seq[String]): Either[String, os.Path] = Right(os.Path(strs.head, os.pwd))
+  }
+
+  @main
+  def run(
+    @arg(name = "mlirbc-file") mlirbcFile: Option[os.Path],
+    @arg(name = "class-name") className: String,
+    @arg(name = "dump-methods") dumpMethods: Flag,
+    @arg(name = "eval") eval: Option[String],
+  ) = {
```

Comment:
I prefer API to be as simple as possible, this API is used by doc/PnR/DV guys. We should statically maintain them.
```
  def dumpOM(
    @arg(name = "mlirbc-file") mlirbcFile: os.Path,
    @arg(name = "class-name") className: String,
  ): String = ???
  def vlen(
    @arg(name = "mlirbc-file") mlirbcFile: os.Path,
    @arg(name = "class-name") className: String,
  ): String = ???
  def dlen(
    @arg(name = "mlirbc-file") mlirbcFile: os.Path,
    @arg(name = "class-name") className: String,
  ): String = ???
```
1. `mlirbcFile`: just use `mlirbcFile`, we generally don't use read from stdin.
2. `className` should be automatically detected, users don't need know what Class is, or force it to be fixed.

Reply:
Makes sense. I have pushed a commit for it, could you review it again please?